Spatial.gd

```diff
@@ -413,13 +420,81 @@ func gen_map_markers():
 		var position_vector = Vector3(position.get_x(), position.get_y(), position.get_z())
 		var texture_path = icon.get_texture_path()
 		if texture_path == null:
-			print("Warning: No texture found for icon")
+			#Some icons have their texture in a parent of the category.
```

Comment:
You have taken a real warning about an error with texture, and removed the warning while the error still occurs. No icons will be rendered still, but we wont know about it anymore.